api/src/models.ts

```diff
@@ -18,7 +18,7 @@ import {
   WpapStatusChoices,
 } from "./choices";
 
-const sequelize = new Sequelize(process.env.DATABASE_URL);
+const sequelize = new Sequelize(process.env.DATABASE_URL, { logging: false });
```

Comment:
This line intends to turn SQL statement logging off: during the integration tests for testing version routing, there's a LOT of output for every time models are looked up in the database.

**question**: Is there a way to set Sequelize to only log errors and warnings, and not debug/info?